framework/utils/InputData.py

```diff
@@ -392,10 +423,9 @@ def handleError(s):
         errorList.append(s)
 
     if node.tag != self.name:
-      #should this be an error or a warning?
+      #should this be an error or a warning? Or even that?
       #handleError('XML node "{}" != param spec name "{}"'.format(node.tag,self.name))
       print('Note: XML node "{}" != param spec name "{}".  This should not usually be an issue.'.format(node.tag,self.name))
```

Comment:
Unfortunately this warning happens for any derived class that uses the "input readers" of the base class...probably should we make this a bit smarter...